components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -163,8 +164,17 @@ async function confirm2() {
   }
 }
 
-watchEffect(() => {
-  price.value = data.value?.offers[0]?.price || ''
+watchEffect(async () => {
+  price.value =
+    currentBlock < data.value?.offers[0]?.expiration
```

Comment:
it should also have an `active` status

Reply:
well it's not rly necesary since it's in the query `where: { status_eq: ACTIVE }`